FogCreek/Glitch-Community
# Change annotation to note, for new v1 route

## Links
* Remix link: https://ancient-salmonberry.glitch.me/ (as of the time I'm making this pr the remix is still building btw it might be faster to pull the branch down locally)
* Issue-tracker link: https://app.clubhouse.io/glitch/story/6230/narrative-notes-not-saving-on-collections

## Changes:
When we switched over to v1 routes it seems we didn't change annotation to note in our requests to update the project in a collection, and unfortunately the backend did not error out on that so it was easy to think it was working (until you refresh the page)

## How To Test:
* on the remix, add or update a note, refresh the page, the note should still be there (you'll notice on prod the note disappears after refresh
